include/ofi_mr.h

```diff
@@ -185,12 +185,14 @@ struct ofi_mr_cache {
 	struct util_domain		*domain;
 	struct ofi_notification_queue	nq;
 	size_t				max_cached_cnt;
+	size_t				max_cached_size;
```

Comment:
Let's set this to a default of SIZE_MAX.  Then we can drop the checks to see if it's non-zero.

Reply:
Ditto